include/roboteam_ai/stp/Play.hpp

```diff
@@ -41,7 +41,7 @@ class Play {
      * if any waiting -> waiting
      * otherwise -> running
      */
-    [[nodiscard]] virtual Status update() noexcept;
+    [[nodiscard]] virtual void update() noexcept;
```

Comment:
void doesn't need nodiscard
```suggestion
    virtual void update() noexcept;
```